bindings/java/src/main/native/jucx_common_def.cc

```diff
@@ -284,6 +289,11 @@ jobject jucx_request_allocate(JNIEnv *env, const jobject callback, ucp_request_p
          env->SetObjectField(jucx_request, request_callback, callback);
     }
 
+    if (memory_type > 0) {
```

Comment:
why check >0 ?

Reply:
For operations that doesn't use memtype (close ep, flush, etc) - passing `-1` as memtype parameter. Because memory_type=0 is a `UCS_MEMORY_TYPE_HOST`

Reply:
maybe pass UCS_MEMORY_TYPE_UNKNOWN to avoid if?